susemanager-utils/susemanager-sls/salt/channels/channels.repo

```diff
@@ -23,7 +23,7 @@ baseurl={{protocol}}://{{hostname}}:{{port}}/rhn/manager/download/{{ chan }}?{{
 {%- endif %}
 type={{ args['type'] }}
 gpgcheck={{ args['gpgcheck'] }}
-repo_gpgcheck={{ args['repo_gpgcheck'] }}
+repo_gpgcheck=0
```

Comment:
sorry, no. This may fix RedHat based repos, but break SUSE.

Please move this check at a place where it is only printed when running on a SUSE system.
E.g. after line 22.